README.md

```diff
@@ -19,8 +19,18 @@ A list of things useful to owners of the GPD Win Max 2
 14. Make sure Dpad is working correctly.
 15. Make sure remote play is working correctly. (A user was having issues with the screen randomly going black and having to force the machine to shut down.)
 ###### Extra things to check for the Win Max 2 2023 refresh.
-1. Functionality of the onboard HDMI port.
-2. If the device is able to run stably with the RAM speed at 7500MT/s
+1. Functionality of the onboard HDMI port (some units arrived with them dead).
+2. If the device is able to run stably with the RAM speed at 7500MT/s.
+3. Check Advanced->AMD PBS->Display Configurations, make sure all is recognized properly.
+4. USB-C display out on both ports (note: hard to diagnose since USB-carried protocols are a mess)
+5. Display having no retention, flickering or being halved.
+6. M.2 2230 SSD and slot not being dead.
+7. Keyboard not being squeaky or otherwise weird.
```

Comment:
not needed, inconsistent spacebars is due to the design and is already addressed in the known issue and workaround section

Reply:
I don't think it's the same issue, as in it's not spacebar only and it's a different kind of issue, at least that's what I understood from the conversations around.